src/components/Loader.js

```diff
@@ -69,21 +88,7 @@ class Loader extends React.Component {
       );
     }
   }
-}
-
-Loader.propTypes = {
-  color: React.PropTypes.string,
-  isLoading: React.PropTypes.bool,
-  isRelative: React.PropTypes.bool,
-  isSmall: React.PropTypes.bool
-};
+});
 
-Loader.defaultProps = {
-  color: StyleConstants.Colors.PRIMARY,
-  isLoading: false,
-  isRelative: false,
-  isSmall: false,
-  children: 'LOADING...'
-};
 
 module.exports = Loader;
```

Comment:
Sorry about that. I always have that issue for some reason even though I switched it in my atom. I usually just go back through on github and fix the changes...